### DISCUSSION TITLE
What does FluxCp do to fetch player counts?

### DISCUSSION THREAD

**Discussion by Syn9673** [rathena/FluxCP#366] [Q&A] [answered]
Hello. I was writing my own control panel with react, and was wondering where does FluxCp get the data for player count? I understand that for online/offline status, it just pings the server. What about player count data?

**Comment by Akkarinage** [comment, marked as answer]
The code starts here: https://github.com/rathena/FluxCP/blob/master/modules/server/status.php#L24

**Reply by Syn9673** [reply]
I assume that rAthena places the online players there?

**Reply by sanasol** [reply]
Yes, athena sets `online = 1` for currently connected/in-game characters in `char` db table.
That's what you should count as online.
